Source/Scene/ModelInstanceCollection.js

```diff
@@ -164,6 +166,14 @@ define([
             get : function() {
                 return this._readyPromise.promise;
             }
+        },
+        shadows : {
+            get : function () { 
+                return this._model.shadows;
+            }
+            set : function () {
+                this._model.shadows = shadows;
```

Comment:
After closer review I noticed that setting the shadows dynamically is actually more involved that setting the model's shadow property. Would you mind removing this getter/setter for now? I'll be happy to merge the rest of the changes and should have a different PR open soon.